docs/api/ShallowWrapper/instance.md

```diff
@@ -2,20 +2,62 @@
 
 Gets the instance of the component being rendered as the root node passed into `shallow()`.
 
-NOTE: can only be called on a wrapper instance that is also the root instance.
+NOTE: can only be called on a wrapper instance that is also the root instance. With React `16.x.x`, `instance()` returns `null` for stateless React component. See example.
 
+#### Returns (React 16.x.x)
```

Comment:
I don't think we need the `.x.x` anywhere; `16.x` should be sufficient (same throughout)